themes/default/content/what-is/what-is-platform-engineering.md

```diff
@@ -79,6 +74,16 @@ Washington Trust Bank modernized its software development and infrastructure pra
 
 ## Choosing an infrastructure as code framework
 
-Choosing an infrastructure as code framework is an important foundation of a platform engineering strategy because it impacts how infrastructure is modeled, tested, distributed, and deployed across an organization. In particular, the choice of IaC language affects infrastructure’s usability, reusability, and capacity for scale. General-purpose languages provide major advantages because they are widely adopted already, have more flexibility and expressiveness than DSLs, and come with rich ecosystems of tools and frameworks that increase productivity. Last but not least, it’s much easier for developers to use IaC when it’s written in languages they already know.
+Given platform engineering's strong focus on infrastructure management, automation, and standardization, the choice of an infrastructure as code framework is particularly important. Platform engineering teams need a solution that enables them to adhere to the principles of platform engineering while empowering their developers and development teams to leverage the cloud in a secure, scalable, reliable, and consistent way.
+
+In looking at the needs of platform engineering teams, five broad areas emerge:
+
+* **Developer control plane**: This area allows platform teams to provide a simple but powerful user experience in a variety of different ways (IDPs, flexible choice of language for IaC artifacts, CLI tools).
+* **Integration and delivery**: Integration with and support for various [CI/CD](/what-is/what-is-ci-cd) tools and platforms (like Kubernetes) are key here. The ability to embed infrastructure as code programs into applications is particularly powerful.
+* **Monitoring and logging**: Native integration with best-in-class observability providers and frameworks is critical.
+* **Security and identity**: Fine-grained access controls, support for policy frameworks and tooling, and centralized access to secrets and configuration information are key facets of this area.
+* **Resources**: Any solution must provide extensive access to all the major providers, platforms, SaaS offerings, and deployment models that teams need for rapid, cloud-based development and deployment.
+
+Pulumi's solution for platform teams encompasses all five of these areas through our core infrastructure as code tool, [Automation API](/automation) for embedding IaC programs into native application code, [Pulumi Deployments](/product/pulumi-deployments) for CI/CD along with support for numerous other CI/CD solutions, [Pulumi ESC](/product/esc) for centralized access to secrets and configuration information, and [CrossGuard](/crossguard) for policy-based controls--including remediation of policy violations---using the same general purpose programming languages that our core IaC offering supports (including TypeScript, Python, Go, C#, Java, and YAML).
```

Comment:
Should we move the product plugs up into the five broad areas bullet points above to have a direct tie in?

Reply:
I've reworked this section, since there were some comments higher up that the "Choosing an infrastructure as code framework" section---which this was located in---didn't feel necessary. Product plugs are now in a "Conclusion" section that includes a revised CTA.